### PR TITLE
PEP8 compliance second part

### PR DESCRIPTION
**Pull request by gyorb**
Refactoring the code is done during the cleanup,
mainly the suppress handling parts

### CODE REVIEW

File: codechecker_lib/option_parser.py [hunk 1]
```diff
@@ -406,14 +408,16 @@ def skip_n(size):
...
+    Requires a full compile command with the compiler, not only arguments.
+    """
 
     # Keep " characters.
     args = args.replace('"', '\\"')
 
     result_map = OptionParserResult()
     for it in OptionIterator(shlex.split(args)[1:]):
-        arg_check(it, result_map) # TODO: do sth at False result, actually skip.
+        arg_check(it, result_map)  # TODO: do sth at False result, skip it
```
Comment (Xazax-hun) [inline codechecker_lib/option_parser.py]: Trailing period removed.

File: codechecker_lib/client.py [hunk 2]
```diff
@@ -41,10 +41,8 @@ def send_suppress(run_id, connection, file_name):
     Collect suppress information from the suppress file to be stored
     in the database.
     """
-    suppress_data = []
-    if os.path.exists(file_name):
-        with codecs.open(file_name, 'r', 'UTF-8') as s_file:
-            suppress_data = suppress_file_handler.get_suppress_data(s_file)
+    suppress_data = suppress_file_handler.SuppressFileHandler. \
```
Comment (bruntib) [inline codechecker_lib/client.py]: Is it on purpose not to use this UTF-8 thing? If it is, then `import codecs` is also unnecessary.

Comment (gyorb) [inline codechecker_lib/client.py]: It should be opened like this because we store user comment here which might contain non-ascii characters.

**Comment by gyorb** [comment]
New pull request was created for pep8 cleanup.

**Pull request closed without merging**